lib/config.js

```diff
@@ -5,24 +5,35 @@ const path = require('path');
 
 /**
  * Handles the selection of the directories.
- * @param {Object} current_window            - The window object of which to start the dialog from
- * @param {Boolean} startup                  - To determinate wheather
- * @return {folder}                          - Returns the folder
+ * @param {Object} current_window     - The window object of which to start the dialog from
+ * @param {Boolean} startup           - To determinate wheather application is invoked while startup or via the settings window
+ * @return {folders}                   - Returns the folders
  */
 function selectFolders(current_window, startup) {
```

Comment:
This whole file should not exist as stated in the previous review. The story should be something like:
1. App loads
2. Uses the package electron-config to make sure the config file exists.
3. If it exists, skip 4 and 5.
4. App creates the file and puts default configurations in it before launching the window-settings.
5. Apps wait for window-settings to be closed to then re-apply the new configurations (if there are any)
6. App does this by using electron-config directly and fetching the values it needs, in this example, the music folders.